tests/test_driver.py

```diff
@@ -28,6 +28,10 @@
 except (FileNotFoundError, AssertionError):
     test_args = None
 
+MINT_ROOT_URL = 'https://mint.intuit.com'
+MINT_ACCOUNTS_URL = 'https://accounts.intuit.com'
+MINT_CREDIT_URL = 'https://credit.finance.intuit.com'
```

Comment:
Can we import these from `api`, or is there a good reason to have them here as well? Just trying to avoid any duplication as we remove some others :)

Reply:
I was trying to build a constants file and pull from there and when that got sticky, I just went to hard code. I didn't even try to reference from API. I'll try that and see!

Reply:
@mrooney there you go, should be better now!